chrome/script.browserify.js

```diff
@@ -160,16 +180,73 @@ function getLoginData() {
   );
 }
 
+function loginToClipboard() {
+  var what = this.what;
+  chrome.runtime.sendNativeMessage(
+    app,
+    {action: "get", entry: this.entry},
+    function(response) {
+      if(chrome.runtime.lastError) {
+        error = chrome.runtime.lastError.message;
+        m.redraw();
```

Comment:
need to make sure `window.close()` is not executed in this case, so users can see an error message